backend/test/test_czi_hosted/unit/common/config/test_server_config.py

```diff
@@ -260,32 +260,35 @@ def test_mulitdatasets_work_e2e(self, mock_render_template):
         assert data_config["config"]["displayNames"]["dataset"] == "pbmc3k"
         assert data_config["config"]["parameters"]["annotations"] is False
         assert data_config["config"]["parameters"]["disable-diffexp"] is False
+
         assert data_config["config"]["parameters"]["about_legal_tos"] == "tos_set1.html"
 
         response = session.get("/set2/pbmc3k.cxg/api/v0.2/config")
 
         data_config = json.loads(response.data)
-        assert data_config["config"]["displayNames"]["dataset"] == "pbmc3k"
-        assert data_config["config"]["parameters"]["annotations"] is True
-        assert data_config["config"]["parameters"]["about_legal_tos"] == "tos_set2.html"
+        self.assertEqual(data_config["config"]["displayNames"]["dataset"], "pbmc3k")
```

Comment:
too bad we don't use pytest!!!

Reply:
Sorry I'm not sure I understand. I think that self.assertEqual works for regular python unittests. I prefer it because when it fails it tells you what didnt match instead of just raising an assertion error

Reply:
I was just lamenting that we still use unittest, instead of pytest. pytest nicely handles asserts (printing all of the useful info) and is generally much nicer than unittest.

Sorry - this was commentary.  I'm going to resolve this, as there is no action on your part!